spring/boot3-actuator-autoconfigure/src/main/java/com/linecorp/armeria/spring/actuate/ArmeriaSpringActuatorAutoConfiguration.java

```diff
@@ -263,7 +279,7 @@ private static void configureExposableWebEndpoint(ServerBuilder sb, @Nullable In
                                          predicate, path, ImmutableMap.of(), cors);
                  });
 
-        if (StringUtils.hasText(endpointMapping.getPath())) {
+        if (StringUtils.hasText(endpointMapping.getPath()) && properties.getDiscovery().isEnabled()) {
```

Comment:
This is not your fault but we also need to check if the `endpointMapping.getPath()` is `/`.
```
When the management context path is set to /, the discovery page is disabled to prevent the possibility of a clash with other mappings.
```

Reply:
Ah. The case seems to have already been taken care of.
It is normalized to "/" -> "" in WebEndpointProperties.
https://github.com/spring-projects/spring-boot/blob/e3aac5913ed3caf53b34eb7750138a4ed6839549/spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/endpoint/web/WebEndpointProperties.java#L69-L74

Reply:
I didn't notice that. 😓 Thanks!